deploy/schemas/build-farm-recipes/aws_ec2.yaml

```diff
@@ -0,0 +1,7 @@
+build_farm_type: str()
```

Comment:
Should this be `str()` or should it be a literal or enum with a single value?  Do we support having two different EC2 build_farm_types in the build_farm_recipes (maybe you would have definitions with different market constraints or one of ondemand and another that is using spot)?

Reply:
Good catch. The schema should enforce the proper build/run farm name. I'll change it to `enum("AWSEC2")` etc.